static/css/style.css

```diff
@@ -1,3 +1,75 @@
+@import url(../third-party/prism/prism.css);
+
+@media (prefers-color-scheme: dark) {
```

Comment:
I'd like to stick with separate files for light and dark. Can we change to:

```css
@import url(light.css) (prefers-color-scheme: light);
@import url(../third-party/prism/default.css) (prefers-color-scheme: light);

@import url(dark.css) (prefers-color-scheme: dark);
@import url(../third-party/prism/dark.css) (prefers-color-scheme: dark);
```

Let's roll back the modifications to the Prism files, as we don't need to modify them with this method, and updating Prism is outside the scope of the PR.